.github/workflows/resources.yml

```diff
@@ -7,10 +7,10 @@ jobs:
     env:
       VERSION: ${{ github.event.action }}
     steps:
+    - uses: actions/checkout@v2
     - name: Read .nvmrc
       run: echo ::set-output name=nvmrc::$(cat .nvmrc)
       id: nvm
-    - uses: actions/checkout@v2
```

Comment:
Haha, good catch! I also found the same issue in the workflow from `web-scrobbler.github.io` (looks like I copied that workflow from that repo). :)